index.js

```diff
@@ -38,12 +68,12 @@ const getCommitMessages = async (repo, sha) => {
 };
 
 const getHeadSha = async (repo, branch = "master") => {
```

Comment:
Wondering if you can rely on `PROJECTS` to use the appropriate branch and remove the need to pass the branch as a variable

Reply:
All projects in `PROJECTS` pass it on explicitly but the script invokes `getHeadSha` on a few more projects outside of the `PROJECTS` global. It's a sensitive default parameter.